server/db/seed.js

```diff
@@ -0,0 +1,61 @@
+var db = require('./db.js');
```

Comment:
Most of these vars should probably be const, if they aren't being reassigned later.